src/setup_cmd.ml

```diff
@@ -49,10 +49,12 @@ let merge =
       ; sprintf "git config --local merge.default 'mergefmt'"
       ]
     in
-    List.iter commands ~f:(fun line ->
+    List.iter
```

Comment:
Opening stdlabel would help reduce the diff

Reply:
I pushed a commit with this change, but the reduction is small.